gms/gossiper.cc

```diff
@@ -1114,6 +1114,10 @@ void gossiper::add_quarantined_host(locator::host_id host_id) {
     logger.debug("Added quarantined host {}", host_id);
 }
 
+bool gossiper::is_quarantined(locator::host_id host_id) const noexcept {
```

Comment:
Doesn't this logic have to be introduced earlier - before we turn off host ID stealing - so that dtests are passing on every commit?

Reply:
No change with this regards in v3

Reply:
Meh, I'm for reviewability here, and we can bisect on PR granularity anyway.